### DISCUSSION TITLE
unstake

### DISCUSSION THREAD

**Discussion by ml1213** [ping-pub/explorer#551] [Blockchains]
hi there! excuse me - can't find how to unstake tokens? (zetachain) where is this unstake button hidden?

thanks

**Comment by liangping** [comment]
unbond on account detail page

**Comment by AhmadBichi** [comment]
hi i unstake my whale but i can't find it in my wallet neither on the website too and the unstaking period has passed please any help

**Reply by liangping** [reply]
Can you check the tx history on other explorers like mintscan.  then you will possibly found the reason.

**Comment by AhmadBichi** [comment]
Thank you for contacting me 
I can’t see any transaction ID on your page I just know that I unstake and the unstaking period is over but nowhere to see my token 

migaloo1ffvz6js5uy4a44wgq2672yx4yffxz4m6vm8alv

Rad. Ahmad Inuwa

**Comment by liangping** [comment]
<img width="1236" alt="image" src="https://github.com/ping-pub/explorer/assets/2882920/3df9863b-1b5a-4843-92c5-8a74806e80ba">

**Comment by liangping** [comment]
it shows your asset are still in lock status.

**Comment by AhmadBichi** [comment]
Thank you for getting back to me but why I can’t see it on the ping page under unbonding token and no where to be found.
However i have undeligate this token for almost 25days but still it’s show 12days more anyway how do i access it on your site soo that i can keep track of the days 
Thank you

Rad. Ahmad Inuwa

**Comment by liangping** [comment]
try to switch a endpoint.

**Comment by AhmadBichi** [comment]
Please can you make it more simpler so that i can understand I don’t understand “end point “

Rad. Ahmad Inuwa
